src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double width;
+    private double height;
+
+    public Rectangle(String color, double width, double height) {
+        super(color);
+        this.width = width;
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
+        return height * width;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Ractangle: width, height:" + width + height + "and area is: "
```

Comment:
The draw method should include the color of the rectangle in the output to fully meet the requirement of printing figure information.